Alignment/OfflineValidation/plugins/DMRChecker.cc

```diff
@@ -1036,8 +1036,7 @@ class DMRChecker : public edm::one::EDAnalyzer<edm::one::SharedResources> {
       //dxy with respect to the primary vertex
       reco::Vertex pvtx;
       edm::Handle<reco::VertexCollection> vertexHandle;
-      reco::VertexCollection vertexCollection;
-      event.getByLabel("offlinePrimaryVertices", vertexHandle);
+      event.getByToken(vertexToken, vertexHandle);
```

Comment:
Thanks for the change! FYI an even more compact and safe version would be 
```C++
      edm::Handle<reco::VertexCollection> vertexHandle = event.getHandle(vertexToken);
```

Reply:
Hi @mmusich do you plan to do this change?

Reply:
not now.

Reply:
@tvami please clarify if the change is necessary for signature. As the code is now, it is buggy (a part from not compliant with the deprecation rules).

Reply:
It's not necessary, I just didnt understand if "not now" means 1-2 hours later, or in a next PR. From this msg I understand it means not in this PR. I'll sign now.

Reply:
addressed at https://github.com/cms-sw/cmssw/commit/309aa0ed5edd8c7502f65b3d8ec4e2743117ecf9